Refresh.GameServer/Extensions/TicketExtensions.cs

```diff
@@ -0,0 +1,31 @@
+using Bunkum.Core;
+using NPTicket;
+using Refresh.GameServer.Authentication;
+
+namespace Refresh.GameServer.Extensions;
+
+public static class TicketExtensions
+{
+    public static TokenPlatform? DeterminePlatform(this Ticket ticket)
+    {
+        if (ticket.SignatureIdentifier == "RPCN" || ticket.IssuerId == 0x33333333)
+            return TokenPlatform.RPCS3;
```

Comment:
The message implied that there could be *more* to the signature than just "RPCN", and implies only the first four bytes of the signature will be "RPCN", are we sure `==` will hold true?

Reply:
Let me ask them.

Reply:
I'm gonna go ahead with "yes this is likely the case" for now but if that changes I'll readdress this.